deploy/manifests/kip/base/cloudinit.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+anka license activate -f $ANKA_LICENSE
+anka license accept-eula || true
```

Comment:
I believe that we need to run those anka commands with `sudo`

Reply:
if it were executed by mac os init it would be executed as a root user